.github/workflows/build-docker-prereq.yml

```diff
@@ -22,7 +22,9 @@ jobs:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
     - name: build image
+      env:
+        DOCKER_HUB_TOKEN: ${{ secrets.DOCKER_HUB_TOKEN }}
       run: |
-        docker build --tag onnxmlirczar/onnx-mlir-llvmimage:initial -f $GITHUB_WORKSPACE/.github/workflows/prereq.Dockerfile $GITHUB_WORKSPACE/utils
-        docker login -u onnxmlirczar -p 143f1da2-332f-45a1-8587-d6cb07c13230 
-        docker push onnxmlirczar/onnx-mlir-llvmimage:initial 
+        docker build --tag $GITHUB_ACTOR/onnx-mlir-llvmimage:initial -f $GITHUB_WORKSPACE/.github/workflows/prereq.Dockerfile $GITHUB_WORKSPACE/utils
```

Comment:
Can you just add as a comment your GitHub conversation:

To make this work on your fork you need to enable the github workflow and set a personal access token in the github secrets for your docker hub account. When the action succeeds it will put the prereq image in your docker hub repository.

Reply:
done